pants.toml

```diff
@@ -2,7 +2,35 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 [GLOBAL]
-pants_version = "2.0.0.dev0"
-v1 = false
-backend_packages = []
-backend_packages2 = []
+pants_version = "2.0.0.dev1"
+
+pythonpath = ["pants-plugins"]
+backend_packages = [
+  # We activate several Python backends so that we can use Pants to format and test our plugin,
+  # even though our src code won't use Python. These backends are not necessary for our plugin to
+  # work.
+  "pants.backend.python",
+  "pants.backend.python.lint.black",
+  "pants.backend.python.lint.docformatter",
+  "pants.backend.python.lint.flake8",
+  "pants.backend.python.lint.isort",
+  # This will activate `pants-plugins/bash/register.py`.
```

Comment:
Might be worth emphasizing that this is so we can *use* the plugin in this repo, which is orthogonal to the fact that it happens to also be implemented in this repo.